internal/testrunner/runners/system/servicedeployer/context.go

```diff
@@ -19,27 +19,12 @@ type ServiceContext struct {
 	// from the Agent container.
 	Ports []int
 
-	// Logs contains folder paths for log files produced by the service.
-	Logs struct {
-		Folder struct {
-			// Local contains the folder path where log files produced by
-			// the service are stored on the local filesystem, i.e. where
-			// elastic-package is running.
-			Local string
-
-			// Agent contains the folder path where log files produced by
-			// the service are stored on the Agent container's filesystem.
-			Agent string
-		}
-	}
```

Comment:
Any particular reason not to keep the same hierarchical structure as before? Just curious if the hierarchy vs. flat is a personal preference or if there's a stronger reason you prefer flat?

Reply:
I kept it flat, because we've only two configuration options. I adjusted it back so it's again structured.